.stylelintignore

```diff
@@ -5,3 +5,4 @@
 /vendor
 /lib/optimizer
 /lib/common
+*.js
```

Comment:
There was something happening at one stage of working on this where camelcase JS variables weren't being allowed in concatenated strings in style props on React components. But it might've just been my editor, so I'm going to commit this and see how it goes.